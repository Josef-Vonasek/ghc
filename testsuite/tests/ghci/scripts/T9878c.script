:l T9878.hs
f
:cd
:m +GHC.StaticPtr
staticPtrKeys
