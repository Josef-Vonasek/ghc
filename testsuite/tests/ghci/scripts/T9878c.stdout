True
Warning: changing directory causes all loaded modules to be unloaded,
because the search path has changed.
[dde4cd622edf824f52f860d40fa20b43]
